### PR TITLE
Commerce 2291 Disable AccountSelector an MiniCart (Minium) if they're on the checkout page

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java [hunk 1]
```diff
@@ -96,8 +98,20 @@ public int doStartTag() {
 
 			putValue("detailsUrl", detailsURL);
 
-			putValue("isDisabled", false);
-			putValue("isOpen", false);
+			boolean disabled = false;
+
+			Layout layout = themeDisplay.getLayout();
+
+			Layout checkoutLayout =
```
Comment (marco-leo) [inline commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java:105]: @alecsloan To identify if exist a checkout page or not we can't do this, use CommerceOrderHttpHelper.getCommerceCheckoutPortletURL

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java [hunk 2]
```diff
@@ -120,6 +120,20 @@ public int doStartTag() {
 
 			putValue(
 				"viewAllOrdersLink", _getViewCommerceOrdersURL(themeDisplay));
+
+			boolean disabled = false;
+
+			Layout layout = themeDisplay.getLayout();
+
+			Layout checkoutLayout =
```
Comment (marco-leo) [inline commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java:128]: Same thing

**Review by marco-leo — CHANGES_REQUESTED** [review]

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java (hunk 1 above)
Comment (alecsloan) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java]: I'll have to make a Util wrapper, that's ok?

Comment (alecsloan) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java:105]: This isn't for checking if the checkout page exists. It's comparing the layout of the current page to the checkout page's. Do you not like that we're explicitly looking for the "/checkout" friendly url?

Comment (ethanbustad) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java:105]: Yeah the "/checkout" URL being the checkout page may not hold true for a custom site using Commerce, so we should do this a different way.

Comment (alecsloan) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java:105]: Where do we stand with this? Is the MiniCart usable on non-minium sites or is it like the Account Selector where we don't really support use on custom sites?

**Comment by marco-leo** [comment]
@alecsloan instead of this, Add new theme settings to indicate if include the account selector and cart in the theme

**Comment by alecsloan** [comment]
@marco-leo changes implemented in #1726

**Pull request closed without merging**